server/utils/users.js

```diff
@@ -0,0 +1,18 @@
+let users = [];
```

Comment:
I would prefer using an object to store users, for performance. This would reduce access time to constant time.